pysoa/common/transport/base.py

```diff
@@ -119,8 +119,18 @@ def receive_response_message(self, receive_timeout_in_seconds=None):
 class ServerTransport(Transport):
     """
     The base server transport defining the interface for transacting PySOA payloads on the server side.
+
+    :param service_name: The name of the service for which this transport will receive requests and send responses
+    :param instance_index: The 1-based index of this process among multiple forks
+    :param metrics: The optional metrics recorder
     """
 
+    def __init__(self, service_name, instance_index=1, metrics=noop_metrics):
```

Comment:
One tiny nit: can you put `instance_index` after `metrics`? Changing the order is still technically breaking for code that instantiates ServerTransport classes providing `metrics` as a positional argument. Putting `instance_index` after `metrics` will fix that potential issue. Other than that, looks good.